swingset-extensions/release-details/index.js

```diff
@@ -28,18 +28,19 @@ function ReleaseDetails({ packageJson = {} }) {
       const [error, registryData] = await response.json()
       if (error) {
         let msg = `Error fetching registry data for ${name}. `
-        msg += `Full error: ${JSON.stringify(error)}`
+        msg += `error.toString(): ${JSON.stringify(error)}`
```

Comment:
Felt this format might be more helpful and transparent for when there are errors.